requirements.txt

```diff
@@ -143,7 +143,7 @@ drf-exceptions-hog==0.3.0
     # via -r requirements.in
 drf-extensions==0.7.0
     # via -r requirements.in
-drf-spectacular==0.21.1
+drf-spectacular==0.24.2
```

Comment:
Python newbie here: Should this not be updated in `requirements.in` as well? My understanding is that we're generating requirements.txt from that file and it's locked there to `0.21.1`.